Tests/UIPilotTests/UIPilotTests.swift

```diff
@@ -1,10 +1,59 @@
+import SwiftUI
 import XCTest
 @testable import UIPilot
 
 final class UIPilotTests: XCTestCase {
-    func testExample() throws {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct
-        // results.
+    
+    enum PageName {
+        case first, second, third
+    }
+    
+    func testPaths() throws {
+        let pilot = UIPilot<PageName>(initial: .first)
+
+        let _ = UIPilotHost(pilot) { route in
+            switch route {
+            case .first: return AnyView(Text("First Page"))
+            case .second: return AnyView(Text("Second Page"))
+            case .third: return AnyView(Text("Third Page"))
+            }
+        }
+        
+        XCTAssertNotNil(pilot.routeMap?(.first))
+        XCTAssertNotNil(pilot.routeMap?(.second))
+        XCTAssertNotNil(pilot.routeMap?(.third))
+
+        XCTAssertEqual(pilot.paths.count, 1)
```

Comment:
Instead of paths, maybe we can test using public API "stack"?

Reply:
yes, the public API `stack` is a better approach.